src/lcos/local/detail/condition_variable.cpp

```diff
@@ -237,45 +229,33 @@ namespace hpx { namespace lcos { namespace local { namespace detail
 
             while (!queue.empty())
             {
-                threads::thread_id_type id = queue.front().id_;
+                hpx::execution::execution_context ctx = queue.front().ctx_;
 
-                queue.front().id_ = threads::invalid_thread_id;
+                // remove item from queue before error handling
+                queue.front().ctx_ = hpx::execution::execution_context();
                 queue.pop_front();
 
-                if (HPX_UNLIKELY(id == threads::invalid_thread_id))
+                if (HPX_UNLIKELY(!ctx))
                 {
                     LERR_(fatal)
                         << "condition_variable::abort_all:"
                         << " null thread id encountered";
                     continue;
                 }
 
-                LERR_(fatal)
-                        << "condition_variable::abort_all:"
-                        << " pending thread: "
-                        << get_thread_state_name(
-                                threads::get_thread_state(id))
-                        << "(" << id << "): "
-                        << threads::get_thread_description(id);
+                //                 LERR_(fatal)
+                //                         << "condition_variable::abort_all:"
+                //                         << " pending thread: "
+                //                         << get_thread_state_name(
+                //                                 threads::get_thread_state(id))
+                //                         << "(" << id << "): "
+                //                         << threads::get_thread_description(id);
```

Comment:
Is this intentional?

Reply:
No, i re-enabled logging again...
This indentation was a result of clang-format.